packages/overlay/src/popper.ts

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+// Bundle only what we want from popper
+// See: https://popper.js.org/docs#popper-lite-tree-shaking
+
+import {
+    popperGenerator,
+    defaultModifiers,
+    Instance,
+    Placement,
+} from '@popperjs/core/lib/popper-lite';
+
+import flip from '@popperjs/core/lib/modifiers/flip';
+import preventOverflow from '@popperjs/core/lib/modifiers/preventOverflow';
+
+export const createPopper = popperGenerator({
+    defaultModifiers: [...defaultModifiers, flip, preventOverflow],
+});
+
+interface WindowNodeWorkaround {
+    process: {
+        env?: {
+            NODE_ENV?: string;
+        };
+    };
+}
+const windowNodeWorkaround = window as WindowNodeWorkaround;
+
+// Popper code references process.env
```

Comment:
This is really annoying. [This bit of code](https://github.com/popperjs/popper.js/blob/master/src/index.js#L102) in the popper source gets built into this in the released npm:

        if (process.env.NODE_ENV !== "production") {

The following hack works around that.

Reply:
I logged this issue: https://github.com/popperjs/popper.js/issues/933

Reply:
Yeah this is because we're not actually bundling our code anywhere, not exactly sure why they have this in their production code tbh, they should probably just remove.

We're running our code through `es-dev-server` though so the simplest fix for us is to do this in `es-dev-server.config.json`:

```
  babelConfig: {
    plugins: ['transform-node-env-inline'],
  },
```

Andd add the babel plugin to the dev dependencies. I think we should probably have `@popperjs/core` as a peer dependency in the package if we do go ahead with this though so it'll probably want to be fixed at source.